Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -367,7 +384,8 @@ private void UpdateThumbStickData(InteractionSourceState interactionSourceState,
                         interactionMapping.Vector2Data = interactionSourceState.thumbstickPosition;
 
                         // If our value changed raise it.
-                        if (interactionMapping.Changed)
+                        if (interactionMapping.Changed ||
+                           !interactionMapping.Vector2Data.Equals(Vector2.zero))
```

Comment:
I'm half tempted to move this into the interactions changed check instead of adding it everywhere in each of our implementations. @SimonDarksideJ thoughts?

Reply:
Nah that doesn't make sense to me. Then "Changed" isn't really what  it is

Reply:
I'm uneasy about this line testing vector2.zero,  As it would mean a value that changed from say 0.1 to 0 wouldn't invoke a changed state, which is has.
We shouldn't ignore 0 as a changed state, I can see where you were going (ignoring if the value remains at 0, but the changed check already does this.

Reply:
The change from 0.1 to 0 would be caught be the first half of the check, because `interactionMapping.Changed` will be true.

Reply:
That was my thought as well.

It's no worries, that's what unit tests are for! woot!